src/main/java/frc/robot/subsystems/drive/Drive.java

```diff
@@ -49,6 +73,15 @@ public void stop() {
         io.setVoltage(0.0, 0.0);
     }
 
+    public void aim() {
+        this.forward = 0;
+        this.rotation = targetHeading.getAsDouble();
+        this.driveArcade(forward, rotation);
+        if (Math.abs(currentHeading.getAsDouble() - targetHeading.getAsDouble()) < 1e-5) {
```

Comment:
This 1e-5 is a very very tight threshold. I would say aim for like 1 degree at first, otherwise we will never be aligned